### ISSUE TITLE
Transfer and modify unit-test for malaria-web app from automated testing

### ISSUE THREAD

**Issue by nfebe** [systers/macc#364]
## Description
As part of the python standards for systers which requires every repository to have and hosts it tests, tests at [ automated-testing/MACCTests2017/tests/unit/test_malaria_web.py ](https://github.com/systers/automated-testing/blob/develop/MACCTests2017/tests/unit/test_malaria_web.py) should be moved into `tests.py` for malaria-web module.

## Acceptance Criteria
- [x] Transfer tests
- [x]  Refactor/Update  tests 
- [x] Ensure tests run and pass successfully

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
6 hours
